UPGRADE.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading
+
+This document describes future changes that will affect your current resources used
+inside of OpenShift. Each change contains description of the change and information
+when that change will happen.
+
+
+## Origin 1.0.0 / OSE 3.0.0
+
+* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
+  All the newly created Builds will already have the new label, old values will
+  be recognized by OpenShift through the 3.0.x release. Support for these values
+  will be removed in OSE 3.1 and Origin 1.x. See #3502.
```

Comment:
the way this is structured it's a little hard to tell what version is impacted...  reading this i might think that "all the new created builds will already have the new label" applies to "3.0.0" which of course it doesn't.

I'd make it more clear what the current behavior is, what the transitional behavior will be,  and what the final behavior will be.

OSE 3.0.0
  currently all build pods have a label named build.  This label is being deprecated in favor of io.openshift.build.name in the X.Y.Z release (update this when we know what release the change is going into) at which point both labels will exist on build pods.  In version 3.1 the old labe (build) will be removed entirely.